src/patternfly/components/Button/examples/buttonStates.hbs

```diff
@@ -1,3 +1,58 @@
-{{#> button btnClass="pf-is-primary pf-has-focus"}}Focused{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-active" btnAriaPressed="true"}}Active{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-disabled" btnTabIndex="-1" btnAriaDisabled="true"}}Disabled{{/button}}
\ No newline at end of file
+<div class="pf-l-level">
+<div class="pf-l-stack">
+    {{#> button btnClass="pf-is-primary"}}
+        Primary
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-focus"}}
+        Primary Focus
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-active" btnAttributes="ariaPressed=\"true\""}}
+        Primary Active
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-disabled" btnAttributes="disabled=\"true\" tabIndex=\"-1\" ariaDisabled=\"true\"" }}
```

Comment:
One of the notes about using ARIA in HTML is that [ARIA adds nothing to the default semantics of most HTML](https://www.w3.org/TR/using-aria/#aria-does-nothing). Since `<button>` natively supports `disabled`, then `aria-disabled` would be considered redundant. 

I did a [quick test page](https://codepen.io/jenngiardino/pen/VxrXrr), and tested in VO on mac Safari, NVDA on FF, and JAWS on Chrome. 

- VO skips the button with `disabled` when using the Tab key to navigate, but will place focus on the button with `aria-disabled`. Both buttons are listed in the rotor menu.
- NVDA also skips the button with `disabled`. I'm not as familiar with NVDA, but couldn't find a dialog that lists form elements.
- JAWS does not skip the button with `disabled`. And both buttons are listed in the dialog that lists form elements.

With either screen reader and either attribute, the button is announced as "unavailable" or "dimmed".

It could be argued that `tabindex="-1"` is redundant when using `disabled`. When using the keyboard to navigate without using a screen reader, `disabled` has the same effect as `tabindex="-1"`. Even though readers were inconsistent in how they responded to the `disabled` with regard to keyboard navigation, using `tabindex="-1"` had no effect on the screen readers.

In this case, I'd suggest using `disabled` and removing `tabindex` and `aria-disabled` attributes.

Reply:
@jgiardino I had included the `tabIndex` and `ariaDisabled` because of the comments already existing in the documentation section. Are the last two sentences before the table no longer accurate? I'll remove them also if so.

Reply:
Oh yeah, good catch. So this is what it is now:
> Disabled buttons should include the aria-disabled="true" attribute to indicate the state of the element to assistive technologies.

> The .pf-is-disabled class uses pointer-events: none to try to disable the link functionality of <a>s, but that CSS property is not yet standardized. In addition, even in browsers that do support pointer-events: none, keyboard navigation remains unaffected, meaning that sighted keyboard users and users of assistive technologies will still be able to activate these links. So to be safe, add a tabindex="-1" attribute on these links (to prevent them from receiving keyboard focus) and use custom JavaScript to disable their functionality.

What about something like this?:
> Disabled buttons should include the `disabled` attribute to indicate the state of the element to assistive technologies.

Oooooh, I'm just now realizing in the second paragraph that it's talking about `<a>` elements. :-/ Yesterday I was testing buttons only. I can test with links when I'm home later to see how they compare, but I'm guessing that they don't support [`disabled`](https://www.w3schools.com/tags/att_disabled.asp). 

Is this example page meant to show examples of the button styles being used for both `<button>` and `<a>` elements? If so, then the second paragraph isn't inaccurate—if we want to simulate a disabled button while using `<a>` then we would need to use the `aria-disabled="true"` and `tabindex="-1"` attributes as described in the second paragraph. I'm inclined to say that we should rely on native semantics when using buttons, and supplement this behavior with aria when using `<a>`, but am interested in what others think.